qmfkdldks/visible-idea-landing
# Library Page

# What is the objective?
Close more deals with the client who need technical support.

- Clients can see how we work
- We need to show the future and dedication
# User story
As a client, I want to check how agency works, so I can decide whether to work with us or not.
# Acceptance criteria
- Show categories
- Show related articles
- It should be carousel (Horizontally scrollable)
- Should show preview when the mouse is hovering

Here is the list

- Form
- Navigation
- Footer
- Carousel
- Animation
- Pagination
# Technical Notes
- List is defined and stored from the visible idea landing project.
- Frontend app should fetch data from contentful
# What should be tested

1. When https://localhost:3000/examples is clicked it should open the page
2. When mouse is hovering the article title, it should show preview image with animation
3. When the article title is clicked, it should open corresponding post (article) https://localhost:3000/article-id
# QA Notes
- Visit https://preview.com/examples
- Hover mouse on any articles and see if it shows preview image.
- When it's clicked, it should open the article
# Engineering Pitfalls
- We need to think about the performance impact when there are too many articles.
# Metrics
How many visitors, visit componet library page?
How many visitors, interact with demo link?
